server/index.js

```diff
@@ -24,6 +110,33 @@ app.get('/api/mainProduct', function(req, res) {
   }).catch((err) => console.log(err, 'err from app.get/api/mainProduct'));
 });
 
+// GET photos data
+app.get('/api/photos', function(req, res) {
+  photos.find()
```

Comment:
the photos should be filtered on the back end to send back photos for one product